ui/src/components/core/combobox/combobox.tsx

```diff
@@ -1,39 +1,60 @@
-import React, { Fragment, useRef, useState } from 'react';
+import React, { Fragment, useEffect, useRef, useState } from 'react';
 import { Combobox as ComboboxLib, Transition } from '@headlessui/react';
-import { Icon } from '@/components/core/icon';
+import { Icon, IconName } from '@/components/core/icon';
 import { Flex } from '@/components/layout';
+import { useDebounce } from '@/hooks/use-debounce';
+import { Separator } from '../separator.styles';
+import { cleanString } from './combobox.utils';
 
 type ComboboxInputProps = {
+  /**
+   * If it's true, the list of options will be displayed
+   */
   open: boolean;
+  /**
+   * Name of the left icon to display in the input
+   */
+  leftIcon: IconName;
+  /**
+   * Function to handle the input change
+   */
   handleInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /**
+   * Function to handle the input click. When the user clicks on the input, the list of options will be displayed
+   */
   handleInputClick: () => void;
 };
 
 const ComboboxInput = ({
   open,
+  leftIcon,
   handleInputChange,
   handleInputClick,
 }: ComboboxInputProps) => (
   <div className="relative w-full cursor-default ">
     <Icon
-      name="search"
+      name={leftIcon}

```

Comment:
Isn't this from the github integration work?

Reply:
I did this changes here cause I was editing the component to allow add options on the combobox list